src/ctap2/attestation.rs

```diff
@@ -318,7 +317,7 @@ impl AsRef<[u8]> for AttestationCertificate {
 }
 
 #[derive(Serialize, Deserialize, PartialEq, Eq)]
-pub struct Signature(#[serde(with = "serde_bytes")] pub(crate) ByteBuf);
+pub struct Signature(#[serde(with = "serde_bytes")] pub Vec<u8>);
```

Comment:
So to be clear, this doesn't have the same problem as `ClientPinResponse.pin_token` had because it's using `serde_bytes`?

Reply:
Yep, the use of both `with = "serde_bytes"` and `ByteBuf` was redundant here. There are tests in `ctap2/commands/make_credentials.rs` that exercise this.